gpg-interface.h

```diff
@@ -3,6 +3,7 @@
 
 #define GPG_VERIFY_VERBOSE	1
 #define GPG_VERIFY_RAW		2
+#define GPG_VERIFY_QUIET	4
 
```

Comment:
If you don't mind, I'll rephrase the commit messages and just post them as comments here.